apps/sources/api/sources/speech/serializers.py

```diff
@@ -0,0 +1,18 @@
+from apps.propositions.api.serializers import PropositionDrfSerializer
+from apps.sources.api.serializers import SourceDrfSerializer
+from apps.sources.models import Speech
+
+
+class SpeechDrfSerializer(SourceDrfSerializer):
+    """Serializer for speech sources."""
+
+    utterance_serialized = PropositionDrfSerializer(read_only=True, source='utterance')
```

Comment:
Even though it could be redundant, would you mind adding comments? e.g.,
```
# `utterance` is just an ID; `utterance_serialized` is the serialized value of the referenced object.
utterance_serialized = PropositionDrfSerializer(read_only=True, source='utterance')
```